packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -71,11 +89,7 @@ storiesOf('Patterns (Blocks)|ContentGroupPictograms', module)
           'Item 1 Copy (required)',
           'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean et ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit sollicitudin, sodales nulla quis, consequat libero.'
         ),
-        cta: object('Item 1 Link', {
-          type: 'local',
-          href: 'https://www.example.com',
-          copy: 'Lorem ipsum dolor',
-        }),
+        cta: toggleCta(boolean('Item 1 Cta', false)),
```

Comment:
Instead of repeating all of this could we just repeat it?

```javascript
const numOfPictograms = number("Pictogram count", 35);
const items = [];

for (let i = 0; i < count; i++) {
  items.push({
    heading: text(`Heading ${i}`, '...');
    copy: text(`Copy ${i}` '...'),
    cta: toggleCta(boolean(`CTA ${i}`, false)),
  });
}
```

Reply:
I've updated it so it looks more like you pointed, Thanks!